R/eval.R

```diff
@@ -208,10 +208,12 @@ evaluate_top_level_expression <- function(exprs,
     }
   }
 
-  ev <- list(value = NULL, visible = FALSE)
-
   if (use_try) {
-    handle <- function(f) try(f, silent = TRUE)
+    handle <- function(code) {
+      tryCatch(code, error = function(err) {
+        list(value = NULL, visible = FALSE)
+      })
```

Comment:
It does seem like it would be easier to understand the return type if this was closer to the withVisible call

Reply:
I'll simplify this further after #153 since it turns out that we don't actually need to capture the visibility of the printed value.